google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -126,9 +127,9 @@
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
don't do `import *`. You can configure this in your intellij, see https://doc.nuxeo.com/corg/intellij-idea-setup/#:~:text=To%20avoid%20IntelliJ%20IDEA%20replacing,Editor%20%3E%20Code%20Style%20%3E%20Java.